src/mock-data.json

```diff
@@ -14,7 +14,7 @@
       "field_modified_date":"2019-10-24T16:00:55-05:00",
       "field_published_date":"2019-10-01T11:20:57-05:00",
       "field_published_date_formatted":"A fortnight ago",
-      "field_tl_": "268:::269:::270:::272:::273:::122:::1118",
+      "field_tl_": "268:::269:::270:::272:::273:::122",
```

Comment:
I think the issue you are seeing is related to this. Syndicated results were previously excluded from the main page results (and placed in the separate syndicated content block). By removing just the tag from these posts, these results are no longer getting excluded from the main page results, the creating offsets. 

I'd say instead, any post that was flagged as syndicated content should be removed from the mock data altogether (as we would no longer be receiving results like this in the API endpoint anyway)